### ISSUE TITLE
[Feature Request]: Custom category colors

### ISSUE THREAD

**Issue by addiebarron** [VueTorrent/VueTorrent#1464]
### Description

When editing a category, allow a custom color to be set for that category. Categories could also be initialized to a random color for easy visual distinguishing at a glance.

### Describe the solution you'd like

In the Edit view for a category, add a color selector. Optionally, add a config option to initialize categories to a random color.

### Describe alternatives you've considered

Custom CSS doesn't help as categories cannot be selected based on content. A userscript would be the only option.

### Complementary informations

- [ ] Is this feature already implemented in the default WebUI?

**Comment by Larsluph** [comment]
Adding an attribute to allow for custom CSS seems like a good quick fix
